src/Submissions.tsx

```diff
@@ -1,59 +1,149 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import {
   Checkbox,
+  Dropdown,
+  Pill,
   Table,
   TableBody,
+  TableDataCell,
   TableHead,
   TableHeader,
   TablePagination,
   TableRow,
 } from "@us-gov-cdc/cdc-react";
 import { Icons } from "@us-gov-cdc/cdc-react-icons";
+import getFileSubmissions from "./utils/api/fileSubmissions";
 
 function Submissions() {
+  const pageLimit = 10;
+  const [currentPageData, setCurrentPageData] = useState([]);
+  const [allData, setAllData] = useState([]);
+
+  useEffect(() => {
+    const fetchCall = async () => {
+      const res = await getFileSubmissions();
+
+      if (res.status != 200) return;
```

Comment:
Not necessary for this ticket, but we should probably add some sort of visual feedback to the user if the api call fails. Some kind of modal, dialog box, spinner, etc.